src/config/external-applications.md

```diff
@@ -47,8 +47,8 @@ Flatpak is another method for installing external proprietary applications on
 Linux. For information on using Flatpak with Void Linux, see the [official
 Flatpak documentation](https://flatpak.org/setup/Void%20Linux/).
 
-> Flatpak's sandboxing will not necessarily protect you from any security and/or
-> privacy-violating features of proprietary software.
+Flatpak's sandboxing will not necessarily protect you from any security and/or
+privacy-violating features of proprietary software.
```

Comment:
`Warning:` ?

Reply:
i'd be happy with "Warning:" here.

Reply:
installation/live-images/prep.md has `**Warning**`, what's your preference?

Reply:
Bolding warnings seems like a good idea.

Reply:
(adding bolded warning here)